Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs

```diff
@@ -256,5 +260,23 @@ public ActionResult ResetSubmissions()
 
             return null;
         }
+
+        public ActionResult RegisterJobForCleaningSubmissionsForProcessingTable()
+        {
```

Comment:
How many times do you need to create the job? If once -> TYou need TODO and issue to remove this method later. Also, pelase log an issue to create TempController

Reply:
I made an issue